Sources/XMTPTestHelpers/TestHelpers.swift

```diff
@@ -11,6 +11,35 @@ import XCTest
 @testable import XMTP
 import XMTPRust
 
+public struct TestConfig {
+    static let TEST_SERVER_ENABLED = _env("TEST_SERVER_ENABLED") == "true"
+    // TODO: change Client constructor to accept these explicitly (so we can config CI):
+    // static let TEST_SERVER_HOST = _env("TEST_SERVER_HOST") ?? "127.0.0.1"
+    // static let TEST_SERVER_PORT = Int(_env("TEST_SERVER_PORT")) ?? 5556
+    // static let TEST_SERVER_IS_SECURE = _env("TEST_SERVER_IS_SECURE") == "true"
+
+    static private func _env(_ key: String) -> String? {
+        ProcessInfo.processInfo.environment[key]
+    }
+
+    static public func skipIfNotRunningLocalNodeTests() throws {
+        try XCTSkipIf(!TEST_SERVER_ENABLED, "requires local node")
```

Comment:
This makes it so that you can run all local-node tests via:
```
$ TEST_SERVER_ENABLED=true swift test
```
(and in CI, once we get the server running, we can also pass in the `env:` to enable them)